sagemaker-pipelines/tabular/caching/sagemaker-pipelines-caching-steps.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>Given that we change the way to get the step arguments, should we update the [<a href="https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-pipelines/tabular/abalone_build_train_deploy/sagemaker-pipelines-preprocess-train-evaluate-batch-transform.ipynb" target="_blank">existing notebook</a> as well?</p><p> </p><p>Here's one example in the existing notebook that we may want to update.</p><pre class="ql-syntax" spellcheck="false">Finally, we take the output of the processor's&nbsp;run&nbsp;method and pass that as arguments to the&nbsp;ProcessingStep. By passing the&nbsp;pipeline_session&nbsp;to the&nbsp;sagemaker_session, calling&nbsp;.run()&nbsp;does not launch the processing job, it returns the arguments needed to run the job as a step in the pipeline.
</pre><p>It's not urgent, we can mark as a follow up item and update after the cache improvement release</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>